src/app/components/product-page/product-page.component.html

```diff
@@ -1,75 +1,81 @@
-<div class="left-side">
-    <h2>
-        {{ product.name }}
-    </h2>
-   
-    <app-rating-stars [rating]="product.rating"></app-rating-stars>
-    <mat-card-subtitle class="comments">
-        Отзывов: {{ product.feedbacksCount }}
-    </mat-card-subtitle>
+<ng-container *ngIf="product$ |async as product; else spinner">
```

Comment:
```suggestion
<ng-container *ngIf="(product$ | async) as product; else spinner">
```
Чуть чуть кодстайл подправить